src/main/scala/com/twitter/finagle/WebSocket.scala

```diff
@@ -2,43 +2,43 @@ package com.twitter.finagle
 
 import com.twitter.finagle.netty3._
 import com.twitter.finagle.param.{Label, ProtocolLibrary, Stats}
-import com.twitter.finagle.client.{Transporter, StdStackClient, StackClient}
-import com.twitter.finagle.server.{Listener, StdStackServer, StackServer}
+import com.twitter.finagle.client.{StackClient, StdStackClient, Transporter}
+import com.twitter.finagle.server.{Listener, StackServer, StdStackServer}
 import com.twitter.finagle.transport.Transport
-import com.twitter.finagle.websocket.{Netty3, ClientDispatcher, Request, Response, ServerDispatcher}
+import com.twitter.finagle.websocket.{ClientDispatcher, Netty3, Request, Response, ServerDispatcher}
 import com.twitter.util.Closable
 import java.net.SocketAddress
+
+import com.twitter.finagle.ssl.client.SslClientConfiguration
 import org.jboss.netty.channel.Channel
 
 object Websocket extends Server[Request, Response] {
-  case class Client(
-    stack: Stack[ServiceFactory[Request, Response]] = StackClient.newStack,
-    params: Stack.Params = StackClient.defaultParams + ProtocolLibrary("ws"))
+  case class Client(stack: Stack[ServiceFactory[Request, Response]] = StackClient.newStack,
+                    params: Stack.Params = StackClient.defaultParams + ProtocolLibrary("ws"))
   extends StdStackClient[Request, Response, Client] {
-    protected type In = Any
-    protected type Out = Any
+    override protected type In = Any
```

Comment:
I believe it's better to use `override`. It adds checks at compile time so you know when method signature changed.